docs/content_model.md

```diff
@@ -104,3 +105,18 @@ The Content Browser enables you to navigate through all content in the site, inc
 You can also use it to search the repository for content, and to create new content when needed.
 
 ![Universal Discovery Widget](img/udw.png "Universal Discovery Widget")
+
+### Content Tree
+
+If you want to navigate through your website with a menu, go to the **Content** tab, select **Content structure**. **Content Tree** in the left sidebar opens expandable content menu of your site.
+
+![Content Tree in the menu](img/left_menu_tree.png "Content Tree in the menu")
+ 
+Unique icons for each Content Type instantly show you what type of content you are selecting. To add a custom icons to your Content Tree follow [configuration tutorial in developer documentation.](https://doc.ezplatform.com/en/latest/guide/extending_ez_platform/#custom-content-type-icons)
+
+Hidden content is blurred out in the tree view. 
+
+To simplify the Content Tree, big lists are shortened with a **Show more** option. You can select it to expand all lists in the tree.
+**Collapse_all** option situated at the bottom of the tree closes all expanded sections.
```

Comment:
```suggestion
The **collapse_all** option situated at the bottom of the tree closes all expanded sections.
```

optional: the suggested change is aimed at maintaining the lowercase of the option name, exactly as in UI